be/src/olap/schema.h

```diff
@@ -144,6 +144,7 @@ class Schema {
     const std::vector<ColumnId>& column_ids() const { return _col_ids; }
     const std::vector<int32_t>& unique_ids() const { return _unique_ids; }
     ColumnId column_id(size_t index) const { return _col_ids[index]; }
+    ColumnId column_id(const std::string& cname) const { return _field_name_to_col_id.at(cname); }
```

Comment:
why need column name here?  column name is not right during light weight schema change. For example, user call alter table rename column

Reply:
fixed